src/components/Activity.vue

```diff
@@ -25,21 +25,37 @@
 			:disable-menu="true"
 			:disable-tooltip="true"
 			:url="activity.icon"
-			:size="15" />
+			:size="20" />
 
 		<div class="activity-entry__content">
-			<RichText class="activity-entry__content__subject" :text="subjectText" :arguments="subjectArguments" />
-			<RichText class="activity-entry__content__message" :text="messageText" :arguments="messageArguments" />
+			<NcRichText class="activity-entry__content__subject" :text="subjectText" :arguments="subjectArguments" />
+			<NcRichText class="activity-entry__content__message" :text="messageText" :arguments="messageArguments" />
 		</div>
 		<span class="hidden-visually">{{ activity.formattedDate }}</span>
 		<span :title="activity.formattedDate" class="activity-entry__date">{{ dateFromNow }}</span>
+		<div v-if="showPreviews" class="activity-entry__preview-wrapper">
+			<component :is="preview.link ? 'a' : 'span'"
+				v-for="preview, index in activity.previews"
+				:key="preview.fileId ?? `preview-${index}`"
+				class="activity-entry__preview"
+				:href="preview.link">
+				<img class="activity-entry__preview-image"
+					:class="{
+						'activity-entry__preview-mimetype': preview.isMimeTypeIcon,
+					}"
+					:src="preview.source"
+					:alt="preview.link ? t('activity', 'Open {filename}', preview) : ''">
```

Comment:
Shouldn't the param be passed as `t('activity', 'Open {filename}', { filename: preview })`?

Reply:
no but I can make it more explicit like this:
`t('activity', 'Open {filename}', { filename: preview.filename })`